docs/daemon/kwild.mdx

```diff
@@ -0,0 +1,152 @@
+
+This section describes multiple ways to initialize the root directory with these configuration files.
+
+### Kwil Admin Tools
+
+The `kwil-admin setup init`command can be used to quickly setup the root directory with the default configuration required for running an isolated single node. Refer to the [setup guide](./../admin/setup#quick-setup-of-a-single-kwil-node) for more information on its usage.
+
+### Manual Configuration
+
+Manually generate and configure private_key, genesis.json and config.toml as shown below.
+
+#### Private Key
+
+Kwil node uses an ed25519 key for all its chain related operations. Refer to the [key management guide](./../admin/keys#generation) for generating node key into `root_dir/private_key`
+
+#### Genesis
+
+Copy the sample [genesis.json](./configuration/genesis.mdx#complete-genesis-configuration) into `root_dir/abci/config/genesis.json` and update the configuration as required.
+
+Manually update the chain_id, initial_height, app_hash configurations in the genesis.json file.
+To start the node as a validator, the node's public key must be listed under the `validators` in the genesis file. 
+
+Refer to the [genesis specification](./../configuration/configuration.mdx) for advanced genesis configuration.
+
+#### Operational Parameters
+
+Copy the sample [config.toml](./configuration/configuration.mdx#sample-configtoml) into `root_dir/config.toml` and update the configuration as required. 
+
+Notable options include:
+  - app.grpc_listen_addr
+  - app.http_listen_addr
+  - chain.rpc.laddr
+  - chain.p2p.laddr
+  - chain.p2p.persistent_peers
+
+Refer to [configuration specification](./../configuration/configuration.mdx) for more information on tuning a Kwil application.
+
+### Override Configuration During Startup
+
+The Kwil application supports the ability to override the configuration at node startup through the command line flags and environment variables
+Use `--help or -h` for detailed information on all the parameters that can be overwritten through command-line flags. 
+
+For instance, to modify the log level and chain RPC address during the node startup, you can use one of the following methods:
+
+```bash title="Environment variable"
+KWILD_LOG_LEVEL=debug
+KWILD_CHAIN_RPC_LISTEN_ADDRESS="tcp://127.0.0.1:26656"
+
+kwild --root_dir "path/to/root/dir"
+```
+
+```bash title="Command-line flags"
+kwild --root_dir "path/to/root/dir" --log.level "debug" --chain.rpc.listen_addr "tcp://127.0.0.1:26656"
+```
+
+```bash title="Both environment and command-line flags"
+KWILD_LOG_LEVEL=debug
+KWILD_CHAIN_RPC_LISTEN_ADDRESS="tcp://127.0.0.1:26656"
+
+kwild --root_dir "path/to/root/dir" --log.level "info" --chain.rpc.listen_addr "tcp://0.0.0.0:26656"
+```
+If both flags and env variables are set, flags take precedence over environment variables. Therefore, in this example, the Kwil application uses info log level and RPC listen address as "tcp://0.0.0.0:26656"
```

Comment:
Just to make this explanation clearer, could you also set the KWILD_CHAIN_RPC_LISTEN_ADDRESS to be something other than `tcp://0.0.0.0:26656`, to show that it would be overwritten?

Also, backticks on the `info` and `tcp://0.0.0.0:26656` please